S/SCIP/build_tarballs.jl

```diff
@@ -49,8 +62,9 @@ products = [
 # Dependencies that must be installed before this package can be built
 dependencies = [
     Dependency(PackageSpec(name="bliss_jll", uuid="508c9074-7a14-5c94-9582-3d4bc1871065")),
-    Dependency(PackageSpec(name="GMP_jll", uuid="781609d7-10c4-51f6-84f2-b8444358ff6d"), v"6.1.2"),
+    Dependency(PackageSpec(name="GMP_jll", uuid="781609d7-10c4-51f6-84f2-b8444358ff6d")),
```

Comment:
This is wrong unless you want this JLL to be only usable in Julia >= 1.6 (and then you really should add a corresponding julia_compat value)

Reply:
got it thank you, reverted